roles/core/precheck/tasks/prepare.yml

```diff
@@ -34,14 +34,14 @@
       notify: reload-sshd
 
     # prepare | Update ListenAddress at sshd_config for daemon network
-    - name: Update ListenAddress at /etc/ssh/sshd_config
+    - name: prepare | Restrict SSH access to admin nodename
       lineinfile:
         path: /etc/ssh/sshd_config
         state: present
         regexp: '#ListenAddress 0.'
         line: 'ListenAddress {{ ansible_fqdn }}'
```

Comment:
As discussed, I guess this should be `scale_admin_nodename`...

Reply:
Achim, 
yes you are right I forgot to change this. 
I will change it